src/java/picard/analysis/CollectGcBiasMetrics.java

```diff
@@ -211,7 +161,7 @@ protected int doWork() {
         for (int i = 0; i < windowsByGc.length; ++i) {
```

Comment:
IntelliJ is warning me that the variable minimumWindowsToCountInSummary is never used despite being calculated. Is that something we need? @yfarjoun